api/openapi.gen.json

```diff
@@ -33,6 +33,48 @@
       }
     },
     "schemas": {
+      "v1.AWSReservation": {
+        "properties": {
+          "amount": {
+            "format": "int32",
+            "type": "integer"
+          },
+          "aws_reservation_id": {
+            "type": "string"
+          },
+          "created_at": {
+            "format": "date-time",
+            "type": "string"
+          },
+          "finished_at": {},
```

Comment:
This is a problem I overlooked earlier, the OpenAPI generator does not support `sql.NullXYZ` structs, I mean it can't.

But this can be mapped nicely because JSON does support `null` natively. We tried to implement our own marshallers but it did not work out so we ended up using separate Payload struct with simply string pointer (here it would be timestamp pointer - perhaps integer) where Go `nil` maps to JSON `null`.

See here:

```go
type AccountPayload struct {
	// Required auto-generated PK.
	ID int64 `json:"id"`

	// Organization ID. Required.
	OrgID string `json:"org_id"`

	// EBS account number. Can be NULL but not blank.
	AccountNumber *string `json:"account_number"`
}
type AccountRequest AccountPayload
type AccountResponse AccountPayload
```

Reply:
And it means I need to create a new payload for the was reservation and the reservation

Reply:
That reads like an unfinished sentence, but that is unfortunately the solution we landed on with @avitova, yes. I do not like it either, but this is exactly why we do actually have payload types - I was expecting this at some point. Came little bit earlier tho :-)